sswlib/src/main/java/components/Mech.java

```diff
@@ -2778,7 +2778,15 @@ public double GetDefensiveFactor() {
         }
 
         MechModifier m = GetTotalModifiers( true, true );
-        retval += m.DefensiveBonus();

```

Comment:
I am concerned about this line since `m.DefensiveBonus()` isn't listed elsewhere in this PR.  I haven't dug in, so I'd like clarification on it's complete removal.  Can you add more detail here?

Reply:
I've looked at it, and from what I can tell `m.DefensiveBonus()` was a flat modifier from components like Void Sigs that didn't take into account the rules on whether the modifier is tied with the TMM. For example, the Archer 9-W has a void sig, and in that example `m.DefensiveBonus()` is 0.3, but it should really be 0.1 because the +3 is tied with the +3 from the TMM.

Are you aware of any other equipment that affects the defensive factor besides voidsig that I can test with?